test/engine_libkbfs.go

```diff
@@ -447,6 +447,11 @@ func (k *LibKBFS) DisableUpdatesForTesting(u User, tlfName string, isPublic bool
 	return nil
 }
 
+// GetConfig implements the Engine interface
+func (k *LibKBFS) GetConfig(u User) libkbfs.Config {
```

Comment:
I think it might be better to avoid exposing `libkbfs.Config`, and instead have a `MakeStaller` function. In theory, a test engine might not even have configs, if it's high-level enough (e.g., a docker engine or something).